README.md

```diff
@@ -24,6 +24,18 @@ jsonQuery.query("$..[?class]", data)
 
 [How to Use](https://dojotoolkit.org/reference-guide/1.10/dojox/json/query.html)
 
+# Additional features #
+JSON Query contains some enhancments not found in the original implementation.
+These include:
+1) Path in results (Currently not supported when running in browser):
+    It is often useful to also recieve the paths to the found items.
+    By setting "pathPropName" when creating the JSON Query object, a property name containing the path will be created in each sub-object of the data.
+    ** NOTE: This operation adds properties to the data
+
+    Example:
```

Comment:
In examples:
we should use 
```js
var jsonQuery = require('json_query');
```
instead of 
```js
var jsonQuery = require('../query');
```

I understand, it was actually my mistake, but we should fix it now

Reply:
> rename `data2` to `data`, more realistic variable name

We already have a var called 'data' there. I don't want to alter the existing tests, just add new ones with more data.

Reply:
> In examples:
> we should use
> 
> ```js
> var jsonQuery = require('json_query');
> ```
> 
> instead of
> 
> ```js
> var jsonQuery = require('../query');
> ```
> 
> I understand, it was actually my mistake, but we should fix it now

Are you talking about the relative path ('..') or the file name?
I'm guessing the file name is query and not json_query because that's the name in dojox.

Reply:
Actually for tests internally we can use `require('../query')` but in read me which is for general public, when we use examples we can use `require('json_query')`

Reply:
`require('../query')`   may confuse general public

Reply:
I changed the readme to ```require('query')```